docs/operations/clean-metadata-store.md

```diff
@@ -143,7 +143,7 @@ Datasource cleanup uses the following configuration:
 
 ### Indexer task logs
 
-You can configure the Overlord to delete indexer task log metadata and the indexer task logs from local disk or from cloud storage.
+You can configure the Overlord to delete indexer task log metadata and the indexer task logs from local disk or from cloud storage.  The cleanup includes the `druid_tasks` and `druid_tasklogs` tables in the metadata database, and the task logs in deep storage.  (Note that `druid_tasklogs` is no longer used and will already be empty, unless the druid version is older.) 
```

Comment:
```suggestion
You can configure the Overlord to delete indexer task log metadata and the indexer task logs from local disk or from cloud storage.
During cleanup, the Overlord removes the `druid_tasks` and `druid_tasklogs` tables from [metadata storage](../configuration/index.md#metadata-storage) and removes task logs from deep storage.
Note that `druid_tasklogs` is no longer used and will already be empty, unless the druid version is older.
```

Reply:
Two follow up questions:
1. Do you know what older version uses `druid_tasklogs`? Wondering if we can say something like "unless the Druid version is XX or older."
2. Should the table be called `druid_taskLog`? That's what's listed in the configuration table: https://druid.apache.org/docs/latest/configuration/index.html#metadata-storage

Reply:
@paul-rogers The cleanup is based on a schedule and is configured by a few different properties. I updated my suggestion based on your recommendation. The properties to configure the cleanup are already in this section in the lines below (https://druid.apache.org/docs/latest/operations/clean-metadata-store.html#indexer-task-logs)

Reply:
Looks good. The following is a bit unclear:

> (Note that `druid_tasklogs` is no longer used and will already be empty, unless the druid version is older.)

Maybe:

> The cleanup includes the `druid_tasks` table in the metadata database, and the task logs in deep storage. In previous versions of Druid task logs were stored in the the metadata database table `druid_tasklogs`. However, this table is no longer used in the current version and the table is always empty.

Also, I like your "periodically" wording so we know that that this isn't one shot:

> You can configure the Overlord to delete indexer logs periodically. Indexer logs include ...

Reply:
Updated.